### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by ghost**
added further explanation on differences between portable installer and regular installer

### CODE REVIEW

File: content/en/docs/refguide/installation/install.md [hunk 1]
```diff
@@ -110,7 +110,9 @@ For more information on how to configure a custom repository, see the *Custom Re
...
 
-As compared to the regular installer, the portable installer installs Mendix Studio Pro in a user-level directory together with all the dependencies. This means that dependencies are not shared between Studio Pro versions, which results in higher disk space usage. In return, you do not require admin rights to install Mendix Studio Pro, while you can still use all the functionality.
+While the regular installer installs Mendix Studio Pro dependencies in system directories, the portable installer installs Mendix Studio Pro in a user-level directory together with all the dependencies. This means that dependencies are not shared between Studio Pro versions, which results in higher disk space usage. In return, you do not require admin rights to install Mendix Studio Pro, while you can still use all the functionality.
+
+The portable installer only includes the necessary tools to build with Studio Pro and run apps locally. It does not contain the MPR tool, mx, mxbuild, or mxuit. You can also not see the Console Log outside of Studio Pro.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/install.md]: @mauritselzinga-mx 
What is `mxuit`? I can't find any reference to it in our documentation.

Also, what do you mean by not being able to see the Console Log outside Studio Pro. I can't see any reference to that in the documentation either.

Comment (ghost) [reply content/en/docs/refguide/installation/install.md]: Hi Mark, 

These are tools that come installed with Studio Pro, or can be leveraged by with Studio Pro. These tools are not mandatory for it's core functionality.

Comment (ghost) [reply content/en/docs/refguide/installation/install.md]: image

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/installation/install.md]: @MarkvanMents , @mauritselzinga-mx , I reviewed changes and tried to make it more clear what tools are included. Could you take a look to make sure that it is technically correct?

File: content/en/docs/refguide/installation/install.md [hunk 2]
```diff
@@ -110,7 +110,9 @@ For more information on how to configure a custom repository, see the *Custom Re
...
 
-As compared to the regular installer, the portable installer installs Mendix Studio Pro in a user-level directory together with all the dependencies. This means that dependencies are not shared between Studio Pro versions, which results in higher disk space usage. In return, you do not require admin rights to install Mendix Studio Pro, while you can still use all the functionality.
+While the regular installer installs Mendix Studio Pro dependencies in system directories, the portable installer installs Mendix Studio Pro in a user-level directory together with all the dependencies. This means that dependencies are not shared between Studio Pro versions, which results in higher disk space usage. In return, you do not require admin rights to install Mendix Studio Pro, while you can still use all the functionalities.
+
+The portable installer does not include all tools that come installed with Studio Pro or can be leveraged by Studio Pro, such as the MPR tool, mx, mxbuild, mxuit, or the Console Log. The portable installer only includes the tools that are necessary to be able to build with Studio Pro and to run apps locally.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/installation/install.md:115]: @MarkvanMents , @mauritselzinga-mx , I reviewed changes and tried to make it more clear what tools are included. Could you take a look to make sure that it is technically correct?

Comment (ghost) [reply content/en/docs/refguide/installation/install.md:115]: This looks good to me.

**Pull request merged**